src/mpi/comm/contextid.c

```diff
@@ -106,6 +105,7 @@ static char *context_mask_to_str(void)
     }
     return bufstr;
 }
+#endif
```

Comment:
Instead of using an unused attribute, I just move the already present preprocessor to include this function, too.